### PR TITLE
Devils Lake data filter

### PR DESCRIPTION
**Pull request by sbmarsh-USGS**
Changed location of if/else statement and created a filter.
#63 fixed.

### CODE REVIEW

File: Report_templates/stagehydrograph.Rmd [hunk 1]
```diff
@@ -18,15 +18,16 @@ wy_start <- wy_dates$wy_start
 wy_end <- wy_dates$wy_end
 
 isDevilsLake <- siteNumber %in% c('05404500')
+startDate <- ifelse(test = isDevilsLake, 
+                      yes = as.Date("1936-01-01"), 
+                      no = as.Date("1800-01-01"))
```
Comment (lindsaycarr) [inline Report_templates/stagehydrograph.Rmd]: this is where we should remove `as.Date`. I think it should just be `yes = "1936-01-01"` and `no = "1800-01-01"`

**Comment by lindsaycarr** [comment]
+ change Middle Genesee file path to be `xlsx` instead of just `xls`

**Comment by sbmarsh-USGS** [comment]
Resaved stagehydrograph.Rmd and pushed it.  Also, the Middle Genesee file path was already fixed.  Should be good to go now.

**Comment by lindsaycarr** [comment]
The change to the file path came through as a commit here, so I wanted to say that on the front page of the PR :)

Good to go! I'll merge

**Pull request merged**